packages/bot-web-ui/src/components/quick-strategy.jsx

```diff
@@ -62,77 +64,141 @@ const QuickStrategyForm = ({
                                     {({ field }) => (
-                                        <Autocomplete
-                                            {...field}
-                                            autoComplete='off'
-                                            className='quick-strategy__duration-dropdown'
-                                            type='text'
-                                            label={localize('Duration unit')}
-                                            list_items={duration_unit_dropdown}
-                                            disabled={duration_unit_dropdown.length === 1}
-                                            onHideDropdownList={() => {
-                                                onHideDropdownList('duration-unit', values[field.name], setFieldValue);
-                                            }}
-                                            onItemSelection={({ value }) => {
-                                                onChangeDropdownItem('duration-unit', value, setFieldValue);
-                                            }}
-                                            onScrollStop={() => onScrollStopDropdownList('duration-unit')}
-                                        />
+                                        <>
+                                            {!is_mobile ? (
+                                                <Autocomplete
+                                                    {...field}
+                                                    autoComplete='off'
+                                                    type='text'
+                                                    label={localize('Duration unit')}
+                                                    list_items={duration_unit_dropdown}
+                                                    disabled={duration_unit_dropdown.length === 1}
+                                                    onHideDropdownList={() => {
+                                                        onHideDropdownList(
+                                                            'duration-unit',
+                                                            values[field.name],
+                                                            setFieldValue
+                                                        );
+                                                    }}
+                                                    onItemSelection={({ value }) => {
+                                                        onChangeDropdownItem('duration-unit', value, setFieldValue);
+                                                    }}
+                                                    onScrollStop={() => onScrollStopDropdownList('duration-unit')}
+                                                />
+                                            ) : (
+                                                <Dropdown
```

Comment:
Ditto here, should use `SelectNative` instead.

Reply:
Same as above.